templates/server/puppetserver/conf.d/puppetserver.conf.erb

```diff
@@ -0,0 +1,60 @@
+# configuration for the JRuby interpreters
+jruby-puppet: {
+    # This setting determines where JRuby will look for gems.  It is also
+    # used by the `puppetserver gem` command line tool.
+    gem-home: /var/lib/puppet/jruby-gems
+
+    # (optional) path to puppet conf dir; if not specified, will use the puppet default
+    master-conf-dir: <%= scope.lookupvar('::puppet::server_dir') %>
+
+    # (optional) path to puppet var dir; if not specified, will use the puppet default
+    master-var-dir: <%= scope.lookupvar('::puppet::vardir') %>
+
+    # (optional) maximum number of JRuby instances to allow
+    max-active-instances: <%= scope.lookupvar('::puppet::server_max_active_instances') %>
+}
+
+# settings related to HTTP client requests made by Puppet Server
+http-client: {
+    # A list of acceptable protocols for making HTTP requests
+    ssl-protocols: [
+    <%- scope.lookupvar('::puppet::server_ssl_protocols').each do |protocol| -%>
+        <%= protocol %>,
+    <%- end -%>
+    ]
+
+    # A list of acceptable cipher suites for making HTTP requests
+    cipher-suites: [
+    <%- scope.lookupvar('::puppet::server_cipher_suites').each do |cipher| -%>
+        <%= cipher %>,
+    <%- end -%>
+    ]
+
+    # The amount of time, in milliseconds, that an outbound HTTP connection
+    # will wait for data to be available before closing the socket. If not
+    # defined, defaults to 20 minutes. If 0, the timeout is infinite and if
+    # negative, the value is undefined by the application and governed by the 
+    # system default behavior. 
+    idle-timeout-milliseconds: <%= scope.lookupvar('::puppet::server_idle_timeout_milliseconds') %>
+
+    # The amount of time, in milliseconds, that an outbound HTTP connection will
+    # wait to connect before giving up. Defaults to 2 minutes if not set. If 0,
+    # the timeout is infinite and if negative, the value is undefined in the
+    # application and governed by the system default behavior.
+    connect-timeout-milliseconds: <%= scope.lookupvar('::puppet::server_connect_timeout_milliseconds') %>
+}
+
+# settings related to profiling the puppet Ruby code
+profiler: {
+    # enable or disable profiling for the Ruby code; defaults to 'false'.
+    enabled: <%= scope.lookupvar('::puppet::server_enable_ruby_profiler') %>
+}
+
+# Settings related to the puppet-admin HTTP API
+puppet-admin: {
+    client-whitelist: [
+    <%- scope.lookupvar('::puppet::server_admin_api_whitelist').each do |certname| -%>
+        <%= certname %>,
```

Comment:
9af8daec579c9db6fd37da289507c78dd21dda82 fixed it for me, puppetserver 1.x expects quotes in the conf file itself.